traits/trait_converters.py

```diff
@@ -42,6 +42,33 @@ def trait_cast(obj):
     return None
 
 
+def as_ctrait(obj):
+    """ Convert to CTrait if the object knows how, else raise TraitError.
+
+    Parameters
+    ----------
+    obj: An object that supports conversion to `CTrait`. Refer
```

Comment:
Please can you update this to follow numpydoc style? https://numpydoc.readthedocs.io/en/latest/format.html